src/Paket.Core/PaketConfigFiles/DependencyCache.fs

```diff
@@ -121,7 +127,11 @@ type DependencyCache (lockFile:LockFile) =
 
             let assemblyFilePerAssemblyDef = 
                 libs |> Seq.map (fun (f:FileInfo) -> 
-                    AssemblyDefinition.ReadAssembly (f.FullName:string), f)
+                    try
+                        (AssemblyDefinition.ReadAssembly (f.FullName:string), f) |> Some
+                    with
+                    | _ -> None)
```

Comment:
same here ;)

Reply:
Yes, sure. Good point.
I was "copying" from https://github.com/fsprojects/Paket/blob/master/src/Paket.Core/Installation/InstallProcess.fs#L300 :)

Reply:
Yes, getting rid of them is hard, because you do not know what you have catched previously ;).
A PR or adding the other case is welcome as well ;)